org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -180,4 +180,16 @@ public void deserialize(final ZigBeeNetworkManager networkState) {
         logger.debug("Loading ZigBee network state: Done.");
     }
 
+    public synchronized void delete() {
```

Comment:
Wouldn't it be worth to contribute this method to the interface in the core framework?

Reply:
See discussion in #222